src/components/Ticketmaster.js

```diff
@@ -96,16 +96,33 @@ const ResultListMobile = Styled.ul`
       }
 `;
 
+const Pagination = Styled.nav`
+  ul {
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+  }
+
+  button {
+    font-size: 1.2rem;
+  }
+`;
+
 const Ticketmaster = function() {
-  const dummy = JSON.parse('{ "_embedded" : { "events": [] }}');
+  const dummy = {
+    _embedded: { events: [] },
+    _links: {},
```

Comment:
this variable is not used?

Reply:
True. I was being explicit, but I could remove any unused variables therein if you like. 🙂